packages/node_modules/@ciscospark/react-hoc-conversation-mercury/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@ciscospark/react-hoc-conversation-mercury",
+  "version": "0.1.0",
```

Comment:
Let's remove versions from the packages `package.json` files. They don't mean anything, but seem to have caused some confusion for devs.